src/webgpu/shader/validation/wgsl/variable-with-initilizer-module-uniform.fail.wgsl

```diff
@@ -0,0 +1,10 @@
+# v-0032: variable 'u' has an initializer, however its storage class is 'uniform'.
+
+[[block]] struct Params {
+  [[offset(0)]] count: i32;
+};
+var<uniform> u : Params = 1; 
```

Comment:
A variable in 'uniform' storage class has to be a struct. Is there a way to init a struct?